MvvmCross/Droid/Droid/Views/Attributes/MvxDialogFragmentPresentationAttribute.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using Android.Views;
+using MvvmCross.Core.Views;
+
+namespace MvvmCross.Droid.Views.Attributes
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = true)]
+    public class MvxDialogFragmentPresentationAttribute : MvxFragmentPresentationAttribute
+    {
+        public MvxDialogFragmentPresentationAttribute()
+        {
+        }
+
+        public MvxDialogFragmentPresentationAttribute(bool cancelable = true, Type activityHostViewModelType = null, bool addToBackStack = false, int enterAnimation = int.MinValue, int exitAnimation = int.MinValue, int popEnterAnimation = int.MinValue, int popExitAnimation = int.MinValue, int transitionStyle = int.MinValue, bool isCacheableFragment = false) : base(activityHostViewModelType, int.MinValue, addToBackStack, enterAnimation, exitAnimation, popEnterAnimation, popExitAnimation, transitionStyle, null, isCacheableFragment)
```

Comment:
Can we put some breaks in here, the ctor lines are suuuuuper long. My screen is not wide enough.